src/Hedgehog/Gen.fs

```diff
@@ -502,15 +502,20 @@ module Gen =
         toRandom g
         |> Random.run seed 30
 
-    let printSample (g : Gen<'a>) : unit =
-        let forest = sampleTree 10 5 g
-        for tree in forest do
-            printfn "=== Outcome ==="
-            printfn "%A" (Tree.outcome tree)
-            printfn "=== Shrinks ==="
-            for shrink in Tree.shrinks tree do
-                printfn "%A" (Tree.outcome shrink)
-            printfn "."
+    let formatSample (gen : Gen<'a>) : string =
```

Comment:
I would expect this to be taking a [format provider](https://docs.microsoft.com/en-us/dotnet/api/system.iformatprovider?view=net-5.0) since it's called format(?) :thinking:

Reply:
&lt;insert joke about naming in computer science here&gt;

We're getting to the point where all we have left are silly names:

- `toString`/`asString`/`stringify`
- `toPretty`/`asPretty`/`prettify`
- `toDisplay`/`asDisplay`/`displayify` (that was intended as a joke)

Do we maybe want to expose a function that uses `IFormatProvider`?

Reply:
How about `render`?

Reply:
`render` is closer to Haskell Hedgehog naming so I think it's a good option.

Reply:
@moodmosaic @TysonMN They are now all a variation of `render`.

Reply:
> How about `render`?

👍